docs/openhabian.md

```diff
@@ -176,7 +176,21 @@ The "Manual/Fresh Setup" submenu entry is the right place for you. Execute all e
 > Please be cautious and have a close look at the console output for errors.
 > Report problems you encounter to the [openHABian Issue Tracker](https://github.com/openhab/openhabian/issues).
 
+
+### YAHM Addon
+
+openHABian can be installed inside a LXC container on Debian/Armbian/Raspbian system supported by [YAHM](https://github.com/leonsio/YAHM). It is irrelevant which SBC is used. During the installation a new minimal [LXC](https://en.wikipedia.org/wiki/LXC) Debian container will be created with openHABian installed there.
```

Comment:
Just so we are clear: The YAHM dependency is self-motivated because of your needs? I like the idea of a container installation but wonder if your script should stay generic without homematic, with the option to also setup homematic. Wdyt?